examples/python/easy_start_demo/client.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
Flowerboxes like this aren't part of the style of the codebase. The timestamp and authorship information is tracked by git. Please feel free to add yourself to the [`AUTHORS` file](https://github.com/grpc/grpc/blob/master/AUTHORS) though.

Reply:
I will remove this part.

Reply:
The flowerbox doesn't appear to be removed.